srcpkgs/sagemath/patches/sympy-use-gmpy2.patch

```diff
@@ -0,0 +1,17 @@
+sagemath is not yet ready to use sympy with flint ground types
+
+With this we force ground types to gmpy2; otherwise we will get
+failures when python3-flint is installed.
```

Comment:
Is this something that could/should be fixed in sympy or python-flint or just something that sagemath needs adjustment for?

Mostly sympy does not expose its use of python-flint directly but the three main exceptions are the domain elements of ZZ, QQ and GF(p).

Reply:
I've no idea. Too many moving parts, python, mpmath, flint, sympy, gmpy2, sagemath, ...